docs/reference/indices/aliases.asciidoc

```diff
@@ -107,7 +107,8 @@ In this case, the alias is a point-in-time alias that will group all
 current indices that match, it will not automatically update as new
 indices that match this pattern are added/removed.
 
-It is an error to index to an alias which points to more than one index.
+Keep in mind that indexing is a single index operation.  Therefore, you will receive an IllegalArgumentException
```

Comment:
I don't think we should document the specific exception type, that's the kind of thing that can easily diverge from the docs over time.